docs/stdlib/numbers.rst

```diff
@@ -251,14 +263,14 @@ from :eql:type:`str` and :eql:type:`json`.
 
     .. note::
 
-        Caution is advised when casting ``bigint`` values into
-        ``json``. The JSON specification does not have a limit on
-        significant digits, so a ``bigint`` number can be losslessly
+        Caution is advised when casting :eql:type:`bigint` values into
+        :eql:type:`json`. The JSON specification does not have a limit on
+        significant digits, so a big integer number can be losslessly
```

Comment:
```suggestion
        significant digits, so a :eql:type:`bigint` number can be losslessly
```

"big integer" feels generic, where `bigint` refers to the type. I think the latter is what we want here.